clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/HttpElasticsearchDAO.scala

```diff
@@ -62,18 +63,18 @@ class HttpElasticsearchDAO private[dataaccess] (
       .query(queryDefinition)
 
     val responsePublisher = httpClient.publisher(searchDefinition)
-    Source.fromPublisher(responsePublisher).map(_.to[D])
+    Source.fromPublisher(responsePublisher).map(_.to[Json].as[D].fold(throw _, identity))
```

Comment:
`.to[D]` -> `.to[Json].as[D]` needed because we no longer store an implicit `HitReader[D]` in the `ElasticsearchIndex`. The goal would be to just to `.to[Json]`, but that starts bleeding into other APIs so I didn't go through with it.